servers/lib/src/files/services/gitlab-files.service.ts

```diff
@@ -21,32 +21,35 @@ export default class GitlabFilesService implements IFilesService {
   }
 
   private async parseArguments(
-    path: string
+    path: string,
   ): Promise<{ domain: string; parsedPath: string }> {
     const gitlabGroup = this.configService.get("GITLAB_GROUP");
     const pathParts: string[] = path.split("/");
     const project: string = pathParts[0];
 
     // Only prepend the gitlabGroup if it's not already part of the path
```

Comment:
gitlab group should never be part of the path sent by user. It is only known from the config file to the code so that the code can prepend the gitlab group before forwarding the query to gitlab instance.

Reply:
I haven't changed that code :no_mouth:

Reply:
Fixed